packages/rich-text-editor/src/RichTextEditor/components/ToggleIconButton/ToggleIconButton.scss

```diff
@@ -41,12 +35,78 @@
   }
 }
 
-.active {
+.default {
+  background-color: $color-white;
+  color: $color-purple-800;
+
+  &:hover:not(.active),
+  &:global(.focus-visible):not(.active) {
+    background-color: $color-gray-200;
+  }
+
+  &.active {
+    background-color: $color-gray-300;
+    color: $color-purple-800;
+  }
+}
+
+.secondary {
+  background-color: $color-white;
+  color: $color-blue-500;
+
+  &:hover:not(.active),
+  &:global(.focus-visible):not(.active) {
+    background-color: $color-blue-100;
+  }
+
+  &.active {
+    background-color: $color-blue-200;
+  }
+}
+
+.primary {
   background-color: $color-blue-500;
   color: $color-white;
+
+  &:hover:not(.active),
+  &:global(.focus-visible):not(.active) {
+    background-color: $color-blue-600;
+  }
+
+  &.active {
+    background-color: $color-blue-700;
+  }
+}
+
+.destructive {
+  background-color: $color-red-500;
+  color: $color-white;
+
+  &:hover:not(.active),
+  &:global(.focus-visible):not(.active) {
+    background-color: $color-red-600;
+  }
+
+  &.active {
+    background-color: $color-red-700;
+  }
+}
+
+.secondary-destructive {
+  background-color: $color-white;
+  color: $color-red-600;
+
+  &:hover:not(.active),
+  &:global(.focus-visible):not(.active) {
+    background-color: $color-red-100;
+  }
+
+  &.active {
+    background-color: $color-red-200;
+  }
 }
 
 .iconWrapper {
-  width: 1rem;
-  height: 1rem;
+  width: 15px;
```

Comment:
Could we do 0.9375rem instead of a px value to make these scale up with font size?

Reply:
Done 👍